core/src/main/scala/org/apache/spark/h2o/utils/H2OSchemaUtils.scala

```diff
@@ -109,26 +108,26 @@ object H2OSchemaUtils {
   }
 
   private def fillArray(
-      elementType: DataType,
-      flatSchemaIndexes: Map[String, Int],
-      buffer: ArrayBuffer[Any],
-      data: Any,
-      qualifiedName: String): Unit = {
+                         elementType: DataType,
```

Comment:
Do you prefer to use rather this formatting? The previous is based on coding conventions from Spark.  I am happy to use any type of formatting, but I would like to agree on which we will use in future.

Reply:
I always use the formatting by Idea after I click on `Reformat code` I don't have strong opinion on the formatting itself, but would like to be able to keep formatting using the Idea.

If there is any template idea can pick up and understand, I'm fine with that :)

Reply:
As we discussed on Slack, we are going to switch to formatting Spark is using, but in the following JIRA, thanks for bringing this up!